source/module.c

```diff
@@ -931,6 +942,46 @@ static void s_handle_failed_callback(napi_env env, napi_value function, napi_sta
     }
 }
 
+napi_status aws_napi_create_external_arraybuffer(
+    napi_env env,
+    void *external_data,
+    size_t byte_length,
+    napi_finalize finalize_cb,
+    void *finalize_hint,
+    napi_value *result) {
+
+    napi_status external_buffer_status =
+        napi_create_external_arraybuffer(env, external_data, byte_length, finalize_cb, finalize_hint, result);
+
+    if (external_buffer_status == NAPI_NO_EXTERNAL_BUFFER_ENUM_VALUE) {
+
+        AWS_LOGF_DEBUG(
+            AWS_LS_NODEJS_CRT_GENERAL,
+            "napi_create_external_arraybuffer (in aws_napi_create_external_arraybuffer) failed with : %s",
+            aws_napi_status_to_str(external_buffer_status));
```

Comment:
don't log this.
If a node installation doesn't support external buffers, this is going to print for every single allocation. It's just too much. Especially at DEBUG level.